tests/test_mgxs_library_condense/results_true.dat

```diff
@@ -1,49 +1,49 @@
-   material  group in nuclide      mean  std. dev.
-0         1         1   total  0.419289    0.01638   material  group in nuclide     mean  std. dev.
-0         1         1   total  0.07774   0.003273   material  group in  group out nuclide      mean  std. dev.
+   group in  material nuclide      mean  std. dev.
```

Comment:
Just curious, any idea why the "group in" and "material" columns are switched now?